data/tlg1799/tlg012/tlg1799.tlg012.1st1K-grc1.xml

```diff
@@ -0,0 +1,1567 @@
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg1799.tlg012.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United States</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Euclidis Opera Omnia</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Henricus Menge</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg1799">Euclid</author>
+                            <imprint>
+                                <publisher>Leipzig</publisher>
+                                <pubPlace>Teubner</pubPlace>
+                                <date>1916</date>
+                            </imprint>
+                            <biblScope unit="volume">8</biblScope>
+                        </monogr>
+                        <ref
+                            target="https://archive.org/stream/euclidisoperaomn08eucluoft#page/n3/mode/2up"
+                            >Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+).(.+)" n="section"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
+                <cRefPattern matchPattern="(.+)" n="chapter"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"
+                />
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="lat">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg1799.tlg012.1st1K-grc1">
+                <div type="textpart" subtype="chapter" n="pr">
+                    <pb n="2"/>
```

Comment:
Since you have "section" as your deepest citation level in one of the chapters, you should go ahead and add a single "section" under each chapter just to keep the citation scheme level, i.e., to make sure that all of the text is at the same depth.

Reply:
Since it's really only one section, I decided it would make more sense to split chapter 13 into a and b instead of adding twenty-something single sections to each chapter. Although it might actually belong to tlg013, since it seems related to "recensio b". But I'm not sure if should split/divide it up like that. This is related to my question to the issue inthe DDD repository.

Reply:
This is a good solution. What was the issue number in the DDD_XML repository?

Reply:
DDD #63, there are overlapping pagenumbers in the spreadsheet and I think it is because of the "recensio b" stuff which woulfd be tlg013.